pom.xml

```diff
@@ -18,6 +18,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <checkstyle.skip>true</checkstyle.skip>
```

Comment:
Drats, you are right.  I added this because we used to have a dep on build-tools in the parent and this caused the initial `install` to fail in travis-ci because during `validate` the jar wouldn't exist.  Now that you changed the paths to the checkstyle config and suppression change I just made, which now are paths to the files versus classpath paths, we don't need to draw an explicit dependency on build-tools in the parent pom.  Anyway, I added this config so we wouldn't get bit by this in the future, but setting it in the parent pom doesn't work since it trickles it to the sub-modules.

See new commit.